po/extra/it.po

```diff
@@ -0,0 +1,69 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.ryonakano.konbucase package.
+# Mirko Brombin <send@mirko.pm>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.ryonakano.konbucase\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-24 15:22+0000\n"
+"PO-Revision-Date: 2020-04-24 15:22+0000\n"
+"Last-Translator: Mirko Brombin <send@mirko.pm>\n"
+"Language-Team: Italian\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.0.6\n"
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:7
+#: data/com.github.ryonakano.konbucase.desktop.in:3
+msgid "KonbuCase"
+msgstr "KonbuCase"
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:8
+#: data/com.github.ryonakano.konbucase.desktop.in:5
+msgid "Convert case in your text"
+msgstr "Converti le maiuscole nel tuo testo"
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:10
+msgid "A small text tool app that allows you convert case in your text."
+msgstr "Un piccolo strumento che permette la conversione delle maiuscole nel tuo testo."
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:13
+msgid "Features include:"
+msgstr "Funzionalità incluse:"
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:15
+msgid ""
+"Click \"Copy to Clipboard\" button to copy the all texts in the text view "
+"without selecting them"
+msgstr ""
+"Clicca il pulsante \"Copia nella Clipboard\" per copiare tutto il testo senza  "
+"selezionarlo"
+
+#: data/com.github.ryonakano.konbucase.appdata.xml.in:16
+msgid ""
+"Convert your text between camelCase, PascalCase, snake_case, kebab-case, and "
+"space-separated"
+msgstr ""
+"Converti il tuo testo tra camelCase, PascalCase, snake_case, kebab-case e ".
```

Comment:
The period after `"` seems to cause the CI failing: https://travis-ci.org/github/ryonakano/konbucase/builds/679028649#L3212. I guess this should be removed.

Reply:
Apparently I entered a dot by mistake. Should be solved now.